packages/shared/getComponentName.js

```diff
@@ -87,7 +91,19 @@ function getComponentName(type: mixed): string | null {
         if (resolvedThenable) {
           return getComponentName(resolvedThenable);
         }
+        break;
       }
+      case REACT_EVENT_COMPONENT_TYPE:
+        return ((type: any): ReactEventComponent).displayName;
```

Comment:
This should probably check for `undefined` before returning, just like the target.  (We probably don't want to _force_ custom event components to have `displayNames`.)